modules/mediaforceBidAdapter.js

```diff
@@ -140,31 +150,47 @@ export const spec = {
           default: return;
         }
       }
-      validImp && imp.push(impObj);
 
-      let request = {
-        id: bid.transactionId,
-        site: {
-          page: referer,
-          ref: referer,
-          id: bid.params.publisher_id,
-          publisher: {
-            id: bid.params.publisher_id
+      let request = requestsMap[bid.params.publisher_id];
+      if (!request) {
+        request = {
+          id: Math.round(Math.random() * 1e16).toString(16),
+          site: {
+            page: referer,
```

Comment:
this new page param value is duplicated by your `ref`, shouldn't `page` be attributed to the window.location.href and the other to the refererInfo property in bidderRequest?

Reply:
Fixed